aliyun/src/test/java/org/apache/iceberg/aliyun/oss/mock/TestLocalAliyunOSS.java

```diff
@@ -0,0 +1,184 @@
+  }
+
+  @After
+  public void after() {
+    OSS_TEST_RULE.tearDownBucket(bucketName);
+  }
+
+  @Test
+  public void testBuckets() {
+    Assert.assertTrue(doesBucketExist(bucketName));
+    assertThrows(() -> oss.createBucket(bucketName), OSSErrorCode.BUCKET_ALREADY_EXISTS);
+
+    oss.deleteBucket(bucketName);
+    Assert.assertFalse(doesBucketExist(bucketName));
+
+    oss.createBucket(bucketName);
+    Assert.assertTrue(doesBucketExist(bucketName));
+  }
+
+  @Test
+  public void testDeleteBucket() {
+    String bucketNotExist = String.format("bucket-not-existing-%s", UUID.randomUUID());
+    assertThrows(() -> oss.deleteBucket(bucketNotExist), OSSErrorCode.NO_SUCH_BUCKET);
+
+    byte[] bytes = new byte[2000];
+    random.nextBytes(bytes);
+
+    oss.putObject(bucketName, "object1", wrap(bytes));
+
+    oss.putObject(bucketName, "object2", wrap(bytes));
+
+    assertThrows(() -> oss.deleteBucket(bucketName), OSSErrorCode.BUCKET_NOT_EMPTY);
+
+    oss.deleteObject(bucketName, "object1");
+    assertThrows(() -> oss.deleteBucket(bucketName), OSSErrorCode.BUCKET_NOT_EMPTY);
+
+    oss.deleteObject(bucketName, "object2");
+    oss.deleteBucket(bucketName);
+    Assert.assertFalse(doesBucketExist(bucketName));
+
+    oss.createBucket(bucketName);
+  }
+
+  @Test
+  public void testPutObject() throws IOException {
+    byte[] bytes = new byte[4 * 1024];
+    random.nextBytes(bytes);
+
+    String bucketNotExist = String.format("bucket-not-existing-%s", UUID.randomUUID());
+    assertThrows(() -> oss.putObject(bucketNotExist, "object", wrap(bytes)), OSSErrorCode.NO_SUCH_BUCKET);
+
+    PutObjectResult result = oss.putObject(bucketName, "object", wrap(bytes));
+    Assert.assertEquals(AliyunOSSMockLocalStore.md5sum(wrap(bytes)), result.getETag());
+  }
+
+  @Test
+  public void testDoesObjectExist() {
+    Assert.assertFalse(oss.doesObjectExist(bucketName, "key"));
+
+    Assert.assertFalse(oss.doesObjectExist(bucketName, "key"));
+
+    byte[] bytes = new byte[4 * 1024];
+    random.nextBytes(bytes);
+    oss.putObject(bucketName, "key", wrap(bytes));
+
+    Assert.assertTrue(oss.doesObjectExist(bucketName, "key"));
+    oss.deleteObject(bucketName, "key");
+  }
+
+  @Test
+  public void testGetObject() throws IOException {
+    String bucketNotExist = String.format("bucket-not-existing-%s", UUID.randomUUID());
+    assertThrows(() -> oss.getObject(bucketNotExist, "key"), OSSErrorCode.NO_SUCH_BUCKET);
+
+    assertThrows(() -> oss.getObject(bucketName, "key"), OSSErrorCode.NO_SUCH_KEY);
+
+    byte[] bytes = new byte[2000];
+    random.nextBytes(bytes);
+
+    oss.putObject(bucketName, "key", new ByteArrayInputStream(bytes));
+
+    byte[] actual = new byte[2000];
+    IOUtils.readFully(oss.getObject(bucketName, "key").getObjectContent(), actual);
+
+    Assert.assertArrayEquals(bytes, actual);
+    oss.deleteObject(bucketName, "key");
+  }
+
+  @Test
+  public void testGetObjectWithRange() throws IOException {
+
```

Comment:
Extra empty line ?

Reply:
Nit: I think this line should be removed because it seems not match the whole iceberg project code style , even if we don't put it into the check style rule set.